_tools/networks.ts

```diff
@@ -153,6 +153,11 @@ export const NETWORKS = [
         url: "https://optimistic.etherscan.io/address/%s",
         source: "directory-ethereum-mainnet-optimism-1.json",
       },
+      {
+        name: "Ethereum Kovan",
+        url: "https://kovan-optimistic.etherscan.io/address/%s",
```

Comment:
I think this should be Optimism Kovan, at least that's how we name all of the other chains

Reply:
Done.